sequencer/worker.go

```diff
@@ -55,21 +56,18 @@ func (w *Worker) AddTxTracker(ctx context.Context, tx *TxTracker) {
 
 		root, err := w.state.GetLastStateRoot(ctx, nil)
 		if err != nil {
-			// TODO: How to manage this
 			log.Errorf("AddTx GetLastStateRoot error: %v", err)
-			return
+			return false
```

Comment:
we are not managing correctly this scenario, if we got an error when getting the nonce/balance and we return from the function, we are not adding the tx to the worker but I understand we are putting the tx as wip in the database. If this is the case, this tx we will not be processed as it doesn't exists for the worker but we are updating in the pool table as wip, therefore this can cause some confusion about the state of this tx. We need to review how to manage this scenario

Reply:
For this one, it would be best to return it to pending status and non-wip. but I'll need to introduce a 2nc return value is_wip.